apps/dashboard/src/app/(dashboard)/dashboard/connect/in-app-wallets/[clientId]/analytics/page.tsx

```diff
@@ -1,12 +1,13 @@
 import type { Range } from "components/analytics/date-range-selector";
 import { InAppWalletAnalytics } from "components/embedded-wallets/Analytics";
 
-export default function Page({
-  params,
-  searchParams,
-}: {
-  params: { clientId: string };
-  searchParams: {
+export default async function Page(props: {
+  params: Promise<{
+    team_slug: string;
+    project_slug: string;
+    clientId: string;
+  }>;
+  searchParams: Promise<{
```

Comment:
ser u didn't check something out / in correctly here - this shouldn't be here :)

Reply:
fixed